pages/terms_of_service.md

```diff
@@ -1,19 +1,19 @@
 # Terms of Service
 
-Last Revised: April 2019
+Last Revised: September 2021
```

Comment:
Should this be November?

```suggestion
Last Revised: November 2021
```

Reply:
@ErikSchierboom yes. Updated to November